app/assets/stylesheets/src/ats/_settings.scss

```diff
@@ -1,15 +1,13 @@
 #settings {
-  .edit_email_template {
```

Comment:
form was changed from `form_for` to `form_with` and there is became no `edit_email_template`